src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -362,15 +420,15 @@ taq deploy nft.tz -e "testing"
 ┌──────────┬──────────────────────────────────────┬───────┬──────────────────┬────────────────────────────────┐
 │ Contract │ Address                              │ Alias │ Balance In Mutez │ Destination                    │
 ├──────────┼──────────────────────────────────────┼───────┼──────────────────┼────────────────────────────────┤
-│ nft.tz   │ KT1PLo2zWETRkmqUFEiGqQNVUPorWHVHgHMi │ nft   │ 0                │ https://ghostnet.ecadinfra.com │
+│ nft.tz   │ KT1SdFLhhL4Z4n4hWoMPxpa1R5LAq25TwQFi │ nft   │ 0                │ https://ghostnet.ecadinfra.com │
 └──────────┴──────────────────────────────────────┴───────┴──────────────────┴────────────────────────────────┘
 ```
 
 ** We have finished the backend! **
```

Comment:
Is this supposed to be bold? Probably not necessary, just summarize what the user did in this section.